src/message/Message.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import 'bulma/css/bulma.css';
+
+
+
+const Message = ({children, isInfo, title }) => {
+    const infoClass = isInfo ? ' is-primary' : '';
+    
+    return (
+            <div className= 'column is-8'>
+            <article className={"message is-primary"+infoClass}>
```

Comment:
The className `is-primary` will repeat here. I don't think that is what you want.

You've already declared a `const infoClass` above which can either be the value of `'is-primary'` or `''`. 
Use only that instead of declaring a static `is-primary` value here:

```suggestion
            <article className={`message ${infoClass}`}>
```